repository/w.json

```diff
@@ -1118,6 +1118,19 @@
 				}
 			]
 		},
+		{
+			"name": "WSL Build",
+			"details": "https://github.com/existentialmutt/wsl_build",
+			"description": "Create custom build systems for projects running in WSL2",
```

Comment:
This seems to match your GitHub repo description, so you can just remove it here (package control will find it in the repo).

Reply:
So, please remove that so we can wrap this up and ship your package!